test.hxml

```diff
@@ -11,4 +11,4 @@
 --remap flash:openfl
 
 ## Neko
--cmd neko build/neko_test.n
\ No newline at end of file
+-neko build/neko_test.n
```

Comment:
Personally I didn't even realized `haxelib run munit test` was an option,
and I don't think people would read the readme either,
so while munit might be more correct I think `-cmd` in test.hxml would work better in practice.